chrome/app.js

```diff
@@ -158,15 +165,15 @@ GmailToTrello.App.prototype.updateData = function() {
     if (self.model.trello.user !== null && self.model.trello.boards !== null) {
         self.popupView.bindData(self.model);
     }
-    self.gmailView.parsingData = false;
-    self.model.gmail = self.gmailView.parseData();
     self.popupView.bindGmailData(self.model.gmail);
 };
 
 GmailToTrello.App.prototype.initialize = function() {
     var self = this;
 
-    this.model.isInitialized = false;
+    self.model.isPopupDataLoaded = false;
+    self.model.isThreadCardsLoaded = false;
+    self.model.init();
```

Comment:
init() is now just authorizing Trello, not making the API requests, so we do it right in the beginning